distribution/src/main/resources/bin/elasticsearch

```diff
@@ -14,21 +14,20 @@
 #
 #   ES_JAVA_OPTS="-Xms8g -Xmx8g" ./bin/elasticsearch
 
+source "`dirname "$0"`"/elasticsearch-env
+
 parse_jvm_options() {
   if [ -f "$1" ]; then
-    echo "$(grep "^-" "$1" | tr '\n' ' ')"
+    echo "`grep "^-" "$1" | tr '\n' ' '`"
```

Comment:
Actually I want to change to `$()` eventually but for now I want consistency while I'm here cleaning up. I'll do `$()` in a follow up.